core/io/file_access.cpp

```diff
@@ -29,9 +29,12 @@
 /**************************************************************************/
 
 #include "file_access.h"
+#include "file_access.compat.inc"
 
+//#region toto
```

Comment:
Debug code?

Reply:
Indeed, I was checking if VS Code supported code region syntax (because I saw some extension mention it, thinking it was a core feature). :stuck_out_tongue: